xml/installation-installation-configure_lbaas.xml

```diff
@@ -232,482 +242,363 @@
   <para>
    If you have already created a network, subnet, router, security settings and
    IPs you can skip the following steps and go directly to creating the load
-   balancers.
+   balancers. In this example we will boot two VMs and then use LBaaS to switch
+   between the "web servers" (a curl scripts). Along the way we will save
```

Comment:
Please don't use inverted commas to describe the web servers. Inverted commas are used to indicate something is untruthworthy, or said with a level of irony (how you'd do "finger air quotes" when speaking to someone). If it is meant to be a name of something, use either <literal> or <command> or anything else. Please change this throughout.

Reply:
I think instead of using "web servers" like this, you're better off explaining what's actually happening here. You're going to boot two VMs and us LBaaS to switch between two pseudo web servers.

I don't know what (a curl scripts) is meant to mean?

Reply:
via curl scripts was supposed to be "via curl scripts", thought I'd fixed that but it must have gotten lost in one of my rebases

Reply:
had removed the other quoted web servers, missed this one, thanks!

Reply:
reworded this a bit in the latest version, please take a look